geometry/polygon_area.cpp

```diff
@@ -0,0 +1,124 @@
+/**
+ * @file
+ * @brief Implementation of [Shoelace
```

Comment:
Is this the implementation of the Shoelace formula or the Polygon Area? 🤔

Reply:
This is a function which uses shoelace formula to calculate the polygon area, much like the jarvis algorithm is used to solve the convex hull problem